hack/start_simulator.sh

```diff
@@ -29,9 +29,11 @@ cleanup_etcd() {
   echo "Clean up finished"
 }
 
-trap cleanup_etcd EXIT
+
+checkEtcdOnPath
```

Comment:
This fun name looks weird. How about use checkIfEtcdExists

Reply:
Better to use check_if_etcd_exists to avoid we mix snake style and camel style in this file.